client/reader/stream/index.jsx

```diff
@@ -437,15 +471,58 @@ class ReaderStream extends Component {
 					renderLoadingPlaceholders={ this.renderLoadingPlaceholders }
 				/>
 			);
+			const sidebarContent = <ReaderSidebarFollowedSites path={ path } />;
+
+			if ( excludesSidebar.includes( streamType ) ) {
+				body = bodyContent;
+			} else if ( wideDisplay ) {
+				body = (
+					<div className="stream__two-column">
+						{ bodyContent }
+						<div className="stream__right-column">{ sidebarContent }</div>
+					</div>
+				);
+				baseClassnames = classnames( 'reader-two-column', baseClassnames );
+			} else {
```

Comment:
That would make some good opportunity for sub-componentization.

Reply:
Mind sharing more around how you would break this out?

Reply:
This is going to get a tad tricky to do now since I already renamed a few components in a child PR of this PR. I have added this to my Third Deploy list of tasks here: https://github.com/Automattic/wp-calypso/issues/67164 I'll circle back and clean this up after I've deployed a couple of other PRs.